packages/drupal/gutenberg_blocks/js/blocks/accordion-item-text.tsx

```diff
@@ -10,8 +10,10 @@ import { PanelBody, SelectControl } from 'wordpress__components';
 
 const { t: __ } = Drupal;
 
-// @ts-ignore
-registerBlockType('custom/accordion-item-text', {
+registerBlockType<{
+  title: string;
+  icon: string;
```

Comment:
Here and in other places where you added typings to `registerBlockType`:

Attributes that have no `default` value should be typed as optional.

For example
```ts
  title?: string;
  icon?: string;
```